src/function.rs

```diff
@@ -215,6 +218,13 @@ impl<'s> FunctionCallbackArguments<'s> {
         .unwrap()
     }
   }
+
+  /// For construct calls, this returns the "new.target" value.
```

Comment:
What value does it return for non-construct calls?

Reply:
`undefined`